user-guide/Basic_Functionality/Visio/tutorials/Using_Visual_Overview_to_switch_spectrum_presets.md

```diff
@@ -0,0 +1,244 @@
+
+1. Right-click the shape data pane and select *Define Shape Data*.
+
+1. In the pop-up window, replace the default "Property1" label with the shape data label `Element` and change the value to the name of the spectrum simulation element you created earlier.
+
+   <!-- Add screenshot of window -->
+
+1. Click the *New* button to add another shape data field with label `Component` and value `Spectrum`.
+
+   <!-- Add screenshot of window -->
+
+1. Click *OK* to close the *Define Shape Data* window.
+
+1. Make sure no more shapes are selected.
+
+1. Save the Visio file.
+
+1. Check the *Visual* page in DataMiner Cube.
+
+The page should now contain a basic spectrum control, with all the features of the spectrum control in the element, but without the ribbon. This control will also display the info pane. In the next step, you will learn how you can keep this from being shown.
+
+## Step 5: Create a preset to hide the info pane
+
+1. In DataMiner Cube, navigate to the *Spectrum Analyzer* data page of your spectrum simulation element.
+
+1. In the *View* tab of the ribbon, click the *Info pane* button to hide the info pane.
+
+1. In the spectrum sidebar on the right, select the *Presets* tab and click the *Manage* button at the bottom.
+
+1. Click the *New* button.
+
+1. Configure the preset:
+
+   - Give the preset the name `NoInfoPane`.
+
+   - Select the *Share preset with other users* checkbox.
+
+   - Click *OK*.
+
+   <!-- Add screenshot of preset configuration before you click OK -->
+
+1. Click *OK* again to close the *Manage presets* pane.
+
+## Step 6: Use the preset on the spectrum component
+
+1. In DataMiner Cube, navigate back to the *Visio spectrum presets* view.
+
+1. Go back to the Visio file you were working on earlier.
+
+   If you have already closed the Visio file, re-open it by right-clicking the *Visual* page on the view card and selecting *Edit in Visio*.
+
+1. Select the rectangular shape that you linked to the spectrum component.
+
+1. Right-click the shape data pane and select *Define Shape Data*.
+
+1. Click *New* to add another shape data field.
+
+1. Configure the new shape data field with the label `ComponentOptions` and the value `preset=NoInfoPane`, and click *OK*.
+
+1. Make sure no more shapes are selected.
+
+1. Save the Visio file.
+
+1. Check the *Visual* page in DataMiner Cube.
+
+You will see that in DataMiner Cube, the spectrum control no longer includes the info pane.
+
+## Step 7: Create presets for the measurements
+
+In this step, you will create additional presets for each of the center frequencies from which the frequency spectrum should be retrieved.
+
+1. In DataMiner Cube, navigate to the *Spectrum Analyzer* data page of your spectrum simulation element.
+
+1. In the *Manual* tab of the spectrum sidebar, change the settings to the desired values. e.g. for a satellite signal this could be around 12GHz center frequency and 200MHz bandwidth.
```

Comment:
I think it would be better to mention the exact parameters and values here, so people who are less familiar with spectrum elements aren't lost in this step.

Reply:
Parameters and screenshots added, replaced "bandwith" with "frequency span" as this could indeed be unclear to users who are not familiar with spectrum jargon.